### ISSUE TITLE
Setting a determined diemnsion for a geometry.

### ISSUE THREAD

**Issue by Yhanwilliams** [feelpp/feelpp#1162]
I am trying to establish a geometry in micrometers however I have been faced with some difficulty.

**Comment by prudhomm** [comment]
is the question related to Feel++?

**Comment by Yhanwilliams** [comment]
No. its not related to feel++. The question is how do I set the units in the geometry of the .geo file in microns or any other desired dimension of length? Also, how do i fix an error using gmsh " identical points in triangulation change element size or mesh random factor", i have tried varying the element size but to no avail, can you help'

**Comment by prudhomm** [comment]
it is up to you to define what is the unit of `1` in the .geo file but then you have to use the proper units for the remaining components of your code
